final_task/rss_reader/rss_reader.py

```diff
@@ -0,0 +1,33 @@
+import os
+import sys
+import logging
+this_dir = os.path.abspath(os.path.dirname(__file__))
+sys.path.append(this_dir)
+from Handler import Handler
+from Logging import logging_decorator
+from ConsoleParse import get_arguments_from_console
+
+
+@logging_decorator
+def main():
+    arg_parser_args = get_arguments_from_console()
+    version = 1.0
+
+    if arg_parser_args.verbose:
+        logging.basicConfig(level=logging.INFO)
+    if arg_parser_args.limit <= 0:
```

Comment:
If limit<=0, you should raise exception, because it's a malformed argument

Reply:
fixed